velox/exec/fuzzer/PrestoQueryRunner.cpp

```diff
@@ -446,7 +447,12 @@ std::optional<std::string> PrestoQueryRunner::toSql(
       }
     }
 
-    appendWindowFrame(functions[i].frame, sql);
+    if (queryRunnerContext_->windowFrames_.find(windowNode->id()) ==
+        queryRunnerContext_->windowFrames_.end()) {
+      appendWindowFrame(functions[i].frame, sql);
+    } else {
+      sql << queryRunnerContext_->windowFrames_.at(windowNode->id()).back();
+    }
```

Comment:
@pramodsatya Could you point me to which unit tests fails? Do they fail because of changes made in this PR?

Reply:
Hi @kagamiori, the failing test is [`toSql` in `PrestoQueryRunnerTest.cpp`](https://github.com/facebookincubator/velox/blob/94dcf028c12a892b753946563edcc4589f69fe79/velox/exec/tests/PrestoQueryRunnerTest.cpp#L179). It fails with the following error:
```
unknown file: Failure
C++ exception with description "unordered_map::at: key not found" thrown in the test body.
```
and it seems to be because the `windowNodeId` is not captured in the map `queryRunnerContext_->windowFrames_`. Before, the `PrestoQueryRunner` was always getting the frame clause from `queryRunnerContext_`.

Reply:
Hi @pramodsatya, this unit test is for the test coverage of PrestoQueryRunner::toSql(). I don't think PrestoQueryRunner::toSql(windowNode) is used anywhere other than the WindowFuzzer. Could you provide this failing test case with queryRunnerContext_->windowFrames_ and move it to a separate unit test for QueryRunnerContext in PrestoQueryRunnerTest.cpp? Thanks!

Reply:
Updated accordingly, thanks!